contracts/pricers/STETHPricer.sol

```diff
@@ -0,0 +1,82 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.6.10;
+
+import {OracleInterface} from "../interfaces/OracleInterface.sol";
+import {OpynPricerInterface} from "../interfaces/OpynPricerInterface.sol";
+import {STETHInterface} from "../interfaces/STETHInterface.sol";
+import {ERC20Interface} from "../interfaces/ERC20Interface.sol";
+import {SafeMath} from "../packages/oz/SafeMath.sol";
+
+/**
+ * @notice A Pricer contract for a Yearn yToken
+ */
+contract STETHPricer is OpynPricerInterface {
+    using SafeMath for uint256;
+
+    /// @notice opyn oracle address
+    OracleInterface public oracle;
+
+    /// @notice stETH token
+    STETHInterface public stETH;
+
+    /// @notice underlying asset for this yToken
+    ERC20Interface public underlying;
+
+    /**
+     * @param _stETH stETH
+     * @param _underlying underlying asset for stETH
+     * @param _oracle Opyn Oracle contract address
+     */
+    constructor(
+        address _stETH,
```

Comment:
Throughout, I think should change to wstETH to make it clear we are collateralizing with wstETH.